trainingRN/src/screens/todo/index.js

```diff
@@ -1,49 +1,65 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import TodoLayout from './layout';
 
-var globalCounter = 3;
+let globalCounter = 3;
 
 const todoList = [
   {
     id: 1,
-    task: 'Homework'
+    task: 'Homework',
+    checked: false
   },
   {
     id: 2,
-    task: 'Study'
+    task: 'Study',
+    checked: false
   },
   {
     id: 3,
-    task: 'Work'
+    task: 'Work',
+    checked: false
   }
-]
+];
 
 class TodoContainer extends Component {
-  state={ todoList: todoList, text: '' }
+  state = { todoList, text: '' };
 
   setNewText = text => {
-    this.setState({ text })
-  }
+    this.setState({ text });
+  };
 
   handleDelete = id => {
-    this.setState(prevState => ({ todoList: prevState.todoList.filter(elem => elem.id !== id)}))
-  }
+    this.setState(prevState => ({ todoList: prevState.todoList.filter(elem => elem.id !== id) }));
+  };
 
   addNewTask = () => {
     globalCounter += 1;
-    const newTask = { id: globalCounter, task: this.state.text }
-    this.setState(prevState => ({ todoList: [...prevState.todoList, newTask] }))
-  }
+    const newTask = { id: globalCounter, task: this.state.text };
```

Comment:
I think this will be fine here, but in other cases, avoid using global variables